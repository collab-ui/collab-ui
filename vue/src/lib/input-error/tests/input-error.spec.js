import { shallowMount } from '@vue/test-utils';
import InputError from '../index.vue';

describe('InputError', () => {
  it('should match snapshot', () => {
    const wrapper = shallowMount(InputError, {
      propsData: {
        error: 'test'
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should match render error with rendered text', () => {
    const container = shallowMount(InputError, {
      propsData: {
        error: 'test'
      }
    });

    expect(container.find('.message').text()).toEqual('test');
  });
});

