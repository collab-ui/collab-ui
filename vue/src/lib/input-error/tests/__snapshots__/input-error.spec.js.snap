// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputError should match snapshot 1`] = `
<div role="alert" class="md-input__messages">
  <div class="message">test</div>
</div>
`;
